edx_proctoring/backends/rest.py

```diff
@@ -332,6 +338,22 @@ def retire_user(self, user_id):
             raise BackendProviderCannotRetireUser(content) from exc
         return data
 
+    def get_onboarding_attempts(self, course_id, **kwargs):
+        url = self.onboarding_statuses_url.format(course_id=course_id)
+        if kwargs:
```

Comment:
I believe we have URL libraries that takes care of `&` and query parameters for you.
Check https://docs.python.org/3/library/urllib.parse.html or other similar libraries